packages/visual-editing/src/ui/Overlays.tsx

```diff
@@ -136,7 +136,11 @@ const OverlaysController: FunctionComponent<{
       } else if (message.type === 'overlay/setCursor') {
         const {element, cursor} = message
 
-        element.style.cursor = cursor
+        if (cursor) {
+          element.style.cursor = cursor
+        } else {
+          element.style.removeProperty('cursor')
```

Comment:
Related to the above: if we don't modify children's `cursor` values, we may not need this `removeProperty`?